conda/recipes/libcudf/meta.yaml

```
@@ -32,7 +32,7 @@ requirements:
     - arrow-cpp 1.0.1.*
     - arrow-cpp-proc * cuda
     - flatbuffers
-    - boost-cpp >=1.72.0
+    - boost-cpp 1.72.0.*
```

Since that's the only one we have. Not aware of any `1.72.0.1` for example.

```suggestion
    - boost-cpp 1.72.0
```